dev/tests/integration/testsuite/Magento/Checkout/_files/guest_quote_with_multiple_addresses_saved.php

```diff
@@ -0,0 +1,61 @@
+<?php
```

Comment:
Please, reuse current fixtures files
You can find examples for all of the cases in

https://github.com/magento/graphql-ce/tree/2.3-develop/dev/tests/api-functional/testsuite/Magento/GraphQl/Quote/Customer

https://github.com/magento/graphql-ce/tree/2.3-develop/dev/tests/api-functional/testsuite/Magento/GraphQl/Quote/Guest
(also in the same testcase class)

Reply:
@naydav  ok. I'll do that. Thanks.